boot/bootchain.go

```diff
@@ -289,8 +289,10 @@ func bootAssetsToLoadChains(assets []bootAsset, kernelBootFile bootloader.BootFi
 // that we do not store the arrays directly as JSON and we can add
 // other information
 type predictableBootChainsWrapperForStorage struct {
-	ResealCount int                   `json:"reseal-count"`
-	BootChains  predictableBootChains `json:"boot-chains"`
+	ResealCount         int                   `json:"reseal-count"`
+	RecoveryResealCount int                   `json:"recovery-reseal-count"`
+	BootChains          predictableBootChains `json:"boot-chains"`
+	RecoveryBootChains  predictableBootChains `json:"recovery-boot-chains"`
```

Comment:
I didn't think of this originally, but probably best is two have two separate files maybe?

Reply:
That sounds like a good idea, I'll explore it. It also should reduce some duplication in data access functions.